themes/shortbread_v1/topics/land.lua

```diff
@@ -112,17 +114,20 @@ themepark:add_proc('area', function(object, data)
         if minzoom then
             a.kind = t.natural
             a.minzoom = minzoom
-        else
+        end
+        if not a.minzoom or a.kind == 'wetland' then
```

Comment:
Not sure I understand this change. How can `a.kind == 'wetland'` here?

Reply:
We want to catch two cases here - objects with no tags except wetland=swamp or another value from shortbread, and objects with natural=wetland and wetland being a value from shortbread.
if t.natural=wetland t.wetland=swamp it will make it to line 118 with a.kind=wetland so it will check the wetland value and set it if appropriate, and if it has only t.wetland=swamp it will also set it.

Shortbread is a bit of a mess here, so I'm trying to pick a reasonable behavior where it doesn't define it. We don't define if the kind of `landuse=grass natural=grassland` should be `grass` or `grassland`. Generally it makes sense to go with the order the tags are listed in, but `wetland=*` is an exception because it normally is accompanied by `natural=wetland`.

We want `natural=wetland` `wetland=swamp` to have `kind=swamp` or we'd never see any `kind=swamp` in practice.

A better way for shortbread to have done this would be to have `kind=wetland` for all of these then sub-class them if the cartographer wanted to tell between types of wetland

Reply:
There is no check for `natural=wetland` at all in the current code. Shortbread doesn't mention that tag, only the `wetland=XXX` tags. So I thinks what's currently implemented fits the shortbread definition. Yes, it is missing about a third of the objects tagged `natural=wetland` because there is not `wetland=...` tag for them, but arguably that's correct by the shortbread definition.